components/invoice.js

```diff
@@ -48,43 +60,31 @@ export default function Invoice ({ invoice, modal, onPayment, info, successVerb,
     variant = 'failed'
     status = 'expired'
     webLn = false
+  } else if (invoice.expiresAt) {
+    variant = 'pending'
+    status = (
+      <CompactLongCountdown
+        date={invoice.expiresAt} onComplete={() => {
+          setExpired(true)
+        }}
+      />
+    )
   }
 
-  useEffect(() => {
-    if (invoice.confirmedAt || (invoice.isHeld && invoice.satsReceived)) {
-      onPayment?.(invoice)
-    }
-  }, [invoice.confirmedAt, invoice.isHeld, invoice.satsReceived])
-
   const { nostr, comment, lud18Data, bolt11, confirmedPreimage } = invoice
 
   return (
     <>
       {webLnError && !(webLnError instanceof WebLnNotEnabledError) &&
-        <div className='text-center text-danger mb-3'>
-          Payment from attached wallet failed:
-          <div>{webLnError.toString()}</div>
+        <div className='text-center fw-bold text-info mb-3' style={{ lineHeight: 1.25 }}>
+          Paying from attached wallet failed:
+          <code> {webLnError.message}</code>
```

Comment:
I think we should begin using red text/backgrounds very sparingly. It should be an absolute emergency.